asmcomp/arm64/emit.mlp

```diff
@@ -468,21 +468,18 @@ module BR = Branch_relaxation.Make (struct
       else if alloc then 3
       else 5
     | Lop (Istackoffset _) -> 2
-    | Lop (Iload  { memory_chunk; addressing_mode; is_atomic }) ->
-      let based = match addressing_mode with Iindexed _ -> 0 | Ibased _ -> 1 in
-      based + begin match memory_chunk with Single -> 2 | _ -> 1 end
-            + (if is_atomic then 1 else 0)
-    | Lop (Istore ( memory_chunk, addressing_mode, initialization )) ->
-      let word_non_init_add =
-        match memory_chunk, addressing_mode, initialization with
-          Word_val, Iindexed ofs, false when ofs != 0 -> 1
-        | Word_val, Ibased (_, ofs), false when ofs != 0 -> 1
+    | Lop (Iload  { memory_chunk; addressing_mode }) ->
```

Comment:
It it correct that we can ignore `is_atomic` here?
In the `Iload` sequence below, we are outputting `dmb ishld` in some cases where `is_atomic` is set.

Reply:
You're right.  Fixed in 8cdbf79 .